Assets/Modules/AI/Scripts/Nodes/Move.cs

```diff
@@ -35,7 +35,7 @@ public override IEnumerator Action()
             float time = 0;
             Vector3 posInit = gameObject.transform.position;
             Vector3 posFinal = posInit;
-            posFinal.x = IsLeft ? posFinal.x + DistToMove : posFinal.x - DistToMove;
+            posFinal.x = IsLeft ? (posFinal.x + DistToMove).Clamp(-3, 3) : (posFinal.x - DistToMove).Clamp(-3, 3);
```

Comment:
pourquoi un clamp(-3,3) ?

Reply:
C'est une sécurité pour ne pas que les ennemis sortent de la map à force d'aller à droite ou a gauche.
Elle était déjà manquante avant

Reply:
> pourquoi elle ferme les yeux quand elle attaque?

Et elle ferme les yeux juste à cause de la résolution du gif. Elle ne ferme pas les yeux in game